Dockerfile

```diff
@@ -8,4 +8,3 @@ ADD https://github.com/rancherio/os/releases/download/${RANCHEROS_VERSION}/vmlin
 ADD https://github.com/rancherio/os/releases/download/${RANCHEROS_VERSION}/initrd /dist/initrd
 
 ENTRYPOINT ["/scripts/lay-down-os"]
-CMD ["/dev/sda"]
```

Comment:
Yeah, can safely remove from here. I'll hold off on passing labels around at this point. We default to the labels in the install script, which line up to the OS defaults. If the OS defaults change, then we can revisit.